src/styles/blocks/catalog.scss

```diff
@@ -22,4 +22,10 @@
       margin-right: 48px;
     }
   }
+
+  &__card:hover {
+    transition-duration: 300ms;
```

Comment:
1) move transition-duration: 300ms; to static state instead of hover one
2) add transition-property to specify what you'd like to change exactly since by default it applies to all
3) move 300ms to a variable